samples/join-a-team-using-qr-code/csharp/README.md

```diff
@@ -0,0 +1,110 @@
+---
+page_type: sample
```

Comment:
Project folder name is still wrong **"join-a-team-using-qr-code"**
Please rename it as "bot-join-a-team-using-qr-code"